wgengine/magicsock/magicsock.go

```diff
@@ -3027,18 +3027,9 @@ func getPeerMTUsProbedMetric(mtu tstun.WireMTU) *clientmetric.Metric {
 	return mm
 }
 
-// GetLastNetcheckReport returns the last netcheck report, running a new one if a recent one does not exist.
+// GetLastNetcheckReport returns the last netcheck report, returning nil if a recent one does not exist.
```

Comment:
I guess it already returned nil in the error case before (even if not documented) so the callers should be ready for that? I guess they're all fine?

Reply:
Yes, exactly